### DISCUSSION TITLE
救命！！！

### DISCUSSION THREAD

**Discussion by Maixiaofanqie** [share121/inter-knot#3209] [其他]
前两天把做不完的暑假作业随手扔空洞里了，但是似乎调查协会有在空洞捡到东西要1物归原主的习惯，有没有好心人帮我把作业毁尸灭迹啊

**Comment by Guessme404** [comment]
不如。。。你直接进入空洞里，他们不就找不到失主了？<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by LT-UT** [comment]
这委托我接了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by LT-UT** [comment]
![image](https://github.com/user-attachments/assets/af127231-758d-42ad-abe2-8d570a94834a)
你作业被尼尼微拿走了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/189d73f83e4892766ac8197f2df156d0_3183903856565806928.png" alt="比利-呜呜呜" width="75" >
